.ci/jobs/e2e-testing-fleet-daily-mbp.yml

```diff
@@ -9,7 +9,7 @@
     scm:
       - github:
           branch-discovery: no-pr
-          head-filter-regex: '(main|8\.\d|7\.17)'
+          head-filter-regex: '(main|8\.\d|7\.17|feature-.*)'
```

Comment:
What effect is this having? It seems like it is just including feature branches in a search?

Reply:
Will enable those branches to be available in the CI pipelines. https://beats-ci.elastic.co/job/e2e-tests/job/e2e-testing-mbp/ and similar multibranch pipelines

In other words, those regex will be the ones scanned and shown in Jenkins